src/pypromice/process/get_l2.py

```diff
@@ -31,10 +31,27 @@ def get_l2(config_file, inpath, outpath, variables, metadata, data_issues_path:
     # Define input path
     station_name = config_file.split('/')[-1].split('.')[0] 
     station_path = os.path.join(inpath, station_name)
+    
+    # checking that data_issues_path is valid
+    if data_issues_path is None:
+        data_issues_path = Path("../PROMICE-AWS-data-issues")
```

Comment:
I only allowed the default relative path to be used in main for temporary backwards compatibility. I would prefer if we don't fall back to assumptions about the deployment environment.
Either we should skip the data corrections if it is not provided or we should raise an exception.

Reply:
I'm moving forward with this version, which is backward compatible and has same behavior of the CLI and python function. We can make input requirements stricter in a subsequent PR.

Reply:
Yes.